services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/cache/PortHostInfoCache.java

```diff
@@ -0,0 +1,73 @@
+package com.futurewei.alcor.dataplane.cache;
```

Comment:
Please add a license statement to each new file.

Reply:
Added.